Tasks/Common/azure-blobstorage-artifactProvider/blobservice.ts

```diff
@@ -13,11 +14,21 @@ export class BlobService {
         this._storageAccessKey = storageAccessKey;
     }
 
-    public async uploadBlobs(source: string, container: string, prefixFolderPath?: string, host?: string): Promise<void> {
+    public async uploadBlobs(source: string, container: string, prefixFolderPath?: string, host?: string): Promise<string[]> {
```

Comment:
what is host? supposed to be

Reply:
I will change this to be taken from VMSS task